inc/admin/ui/meta-boxes.php

```diff
@@ -88,8 +88,8 @@ function rocket_display_cache_options_meta_boxes() {
 				?>
 
 				<input name="rocket_post_exclude_hidden[<?php echo esc_attr( $field ); ?>]" type="hidden" value="on">
-				<input name="rocket_post_exclude[<?php echo esc_attr( $field ); ?>]" id="rocket_post_exclude_<?php echo esc_attr( $field ); ?>" type="checkbox"<?php echo $title; ?><?php echo $checked; ?><?php echo $disabled; ?>>
-				<label for="rocket_post_exclude_<?php echo esc_attr( $field ); ?>"<?php echo $title; ?><?php echo $class; ?>><?php echo $label; ?></label><br>
+				<input name="rocket_post_exclude[<?php echo esc_attr( $field ); ?>]" id="rocket_post_exclude_<?php echo esc_attr( $field ); ?>" type="checkbox"<?php echo esc_attr( $title ); ?><?php echo esc_attr( $checked ); ?><?php echo esc_attr( $disabled ); ?>>
```

Comment:
Take a look at $title, $checked and $disabled vars in here:
https://github.com/wp-media/wp-rocket/blob/d3696c7a5614eaf8473d92c9bcffe88e8fb456cc/inc/admin/ui/meta-boxes.php#L83-L87

`$disabled` and `$checked` are used with `$echo = false`
I don't think in this case will return HTML content inside.

`$title` is already escaped on line 85

Reply:
>`$disabled` and `$checked` are used with `$echo = false`
I don't think in this case will return HTML content inside.

`$echo = false` means the functions will return HTML instead of echoing it. So the vars will contain html content.

From the [documentation](https://wpseek.com/function/disabled/): ```(bool) $echo Whether to echo or just return the string```

>`$title` is already escaped on line 85

Indeed, so there is no need to escape it again on line 91 (moreover, the whole thing this time).